src/main/deltachat/chatlist.js

```diff
@@ -36,9 +36,10 @@ module.exports = class DCChatList extends SplitOut {
       chatList.push(id)
       if (id === chatId) i = counter
     }
-    if (i === -1) return
-    const chat = this.getSmallChatById(chatId, list, i)
-    this._controller.sendToRenderer('DD_EVENT_CHAT_MODIFIED', { chatId, chat })
+    if (i === -1) return undefined
+    // const chat = this.getChatListItemById(chatId, list, i) <- this should never be called
+    // this._controller.sendToRenderer('DD_EVENT_CHAT_MODIFIED', { chatId, chat })
+    // did we mean to send the full chat?
```

Comment:
Where do we return the chatList? Is this pr done?

Reply:
returns undefined, because otherwise standard complains.
I left the invalid code there (commented out), because maybe you still need it and want to fix it using the full chat object.

Reply:
But we don't send anything to the frontend anymore.

Reply:
no, we don't I assume. I could change that before merge this (using the complete chat object.) should I solve it that way?

Reply:
You removed the line sending the object to the frontend, so i guess you should add it back :D Sorry i commented on the wrong line, you removed it on line 41 https://github.com/deltachat/deltachat-desktop/pull/1131/files/b22982600f8acd41fe52e0273e0dab9dc49a455b#diff-a34cb1175ba7e32523eea0092de1dfc6L41